src/config/overrides/Workspaces.js

```diff
@@ -228,7 +248,32 @@ export const WORKSPACES = [
                   queryId: 'customers',
                 },
               ],
-              attributes: ['Satisfaction', 'SurroundingSatisfaction', 'Thirsty'],
+              attributes: ['Name', 'Satisfaction', 'Surrounding', 'Thirsty'],
+              previewTable: {
+                columns: [
+                  {
+                    field: 'name',
```

Comment:
The case of field names is not consistent between bar and customer attributes (`NbWaiters`, `RestockQty`, `Stock` vs. `name`, `satisfaction`, ...). Could we change it to make it more consistent?